api/include/can_protocols/fault_can_protocol.h

```diff
@@ -1,6 +1,6 @@
 /**
  * @file fault_can_protocol.h
- * @brief Fault CAN Protocol.
+ * @brief Global CAN Protocol.
```

Comment:
Is this comment still correct?

Reply:
This looks questionable.  If this file is still all fault stuff this needs to be changed back.  If its no longer just fault stuff we should change the file name.

Reply:
It will go to global in the runtime config PR but yeah not sure how that got in here I'll change it back for posterity.